### DISCUSSION TITLE
activate a GPIO output

### DISCUSSION THREAD

**Discussion by mathiasonia** [photobooth-app/photobooth-app#237] [Ideas]
Hi

now my photobooth is OK but before this version I used a camera as a photobooth on a tv screen but there is always a lack of luminosity.

now I ask if it's possible to activate a PIN when the picture is take and this one can drive a led or a professionnal flah device or relay or what you want

@mgineer85 you speak about this possibility

"https://github.com/photobooth-app/photobooth-app/discussions/98#discussioncomment-7362091"

**Comment by mgineer85** [comment]
Hi,
I tried to understand the problem. Actually I think the solution can be easy: Use a permanent light. This is also good idea because the liveview would look proper already.

The GPIO of the PI is probably not possible to sync and use for a trigger to a professional flash anyways.  The trigger has to come from the camera itself.
